catroid/src/main/java/org/catrobat/catroid/pocketmusic/ui/TactScrollRecyclerView.java

```diff
@@ -152,6 +164,12 @@ public void onBindViewHolder(TactViewHolder holder, int position) {
 
 		@Override
 		public int getItemViewType(int position) {
+			if (isPlaying) {
+				if (position >= getItemCount() - 2) {
```

Comment:
I would suggest to replace the 2 with a constant named something like EMPTY_VIEW_COUNT_ON_END